src/main/proto/spine/text/text.proto

```diff
@@ -43,15 +43,39 @@ message Text {
 // A position in the text.
```

Comment:
We need some introduction into `beyond_text` thing. Since we now have imaginary position, it should be explained first why we want to have it at all, and then go down to the implementation details in fields.